src/AddIns/Uno.UI.Lottie/ts/Uno.UI.Lottie.ts

```diff
@@ -266,7 +266,17 @@ namespace Uno.UI {
 			if (Lottie._player) {
 				action(Lottie._player);
 			} else {
-				require([`${config.uno_app_base}/lottie`], (p: LottiePlayer) => {
+				if (typeof require !== "function") {
+					console.error("RequireJS not present.");
+					return;
+				}
+
+				const lottieDependencyName = config.uno_dependencies.find((d: string)=> d.slice(-7) === "/lottie");
```

Comment:
`-7` ? :)

Reply:
> `-7` ? :)

That's the equivalent of [`.EndsWith()`](https://learn.microsoft.com/en-us/dotnet/api/system.string.endswith) in C#

Reply:
Why not use TS `String.endsWith()` ? I mean, this works, but it might not be obvious.